src/main/webapp/index.html

```diff
@@ -98,12 +98,15 @@
    opacity: 0.9;
  }
 
+[v-cloak] {
+  display: none;
+}
```

Comment:
The vue loading is actually pretty fast? It's the page loading that takes a bit longer. But since you can see the page loading maybe that's not necessary. We can discuss this in the meeting.

Reply:
Fine. 
I have added a progress loader on another page, but it didn't work significantly.